e2e/deleteapi_test.go

```diff
@@ -16,9 +16,13 @@ func TestDeleteAPI(t *testing.T) {
 
 	// delete any api-server pods
 	deletedPods := make(map[string]struct{})
-	for _, pod := range apiPods.Items {
+	for i, pod := range apiPods.Items {
 		if err := client.CoreV1().Pods("kube-system").Delete(pod.ObjectMeta.Name, &metav1.DeleteOptions{}); err != nil {
-			t.Fatalf("error deleting api-server pod: %v", err)
+			if i == 0 {
+				t.Fatalf("error deleting api-server pod: %v", err)
+			} else {
+				t.Logf("error deleting api-server pod: %v", err)
```

Comment:
Should we add a retry instead of just logging the error? We still want to delete all the pods right?

Reply:
I would try to catch the specific `http2` error we are seeing and retry on that.